src/views/credits/credits.jsx

```diff
@@ -185,64 +187,22 @@ var Credits = React.createClass({
                     </li>
                 </ul>
 
-                <h2>Previous MIT Scratch Team Members</h2>
-                <p>
-                    Many important contributions have been made by previous Scratch Team members,
-                    including John Maloney (who led software development for the first decade of Scratch),
-                    Andrés Monroy-Hernández (who led the development of the first Scratch community website),
-                    Amos Blanton, Champika Fernando, Shane Clements, Abdulrahman idlbi, Evelyn Eastmond,
-                    Amon Millner, Eric Rosenbaum, Jay Silver, Karen Brennan, Leo Burd, Oren Zuckerman, Gaia Carini,
-                    Michelle Chung, Margarita Dekoli, Dave Feinberg, Megan Haddadi, Chris Graves, Tony Hwang, Di Liu,
-                    Tammy Stern, Lis Sylvan, Claudia Urrea, Ben Berg, Hannah Cole, Andrea Saxman, and Randy Jou.
-                </p>
-
-                <h2>Design and Development Partners</h2>
-                <p>
-                    Paula Bontá and Brian Silverman, Playful Invention Company (who started contributing to the
-                    design of Scratch even before it was called Scratch).
-                </p>
+                <h2><FormattedMessage id='credits.previousTitle' /></h2>
+                <p><FormattedMessage id='credits.previousBody' /></p>
 
-                <h2>Scratch Researchers</h2>
-                <p>
-                    <a href="https://scratch.mit.edu/info/research/">Research on Scratch </a>
-                    is being conducted by members of the MIT Scratch Team and researchers at
-                    other universities, including Yasmin Kafai (who collaborated on the
-                    <a href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=0325828"> initial NSF Scratch grant</a>
-                    ) at the University of Pennsylvania Graduate School of Education, Karen Brennan (who leads the
-                    <a href="http://scratched.gse.harvard.edu/"> ScratchEd project</a>) at the Harvard Graduate School
-                    of Education, Benjamin Mako Hill at the University of Washington, Andrés Monroy Hernández at
-                    Microsoft Research, Mimi Ito and Crystle Martin at the University of California, Irvine, Quinn
-                    Burke at College of Charleston, Deborah Fields at Utah State University, and Kylie Peppler at
-                    Indiana University.
-                </p>
+                <h2><FormattedMessage id='credits.partnersTitle' /></h2>
+                <p><FormattedMessage id='credits.partnersBody' /></p>
```

Comment:
I don't think it has links in it? https://github.com/LLK/scratch-www/pull/1148/files#diff-5f47088cec7083daba3b48c14ae030e4R9